googlechrome-90.json

```diff
@@ -0,0 +1,51 @@
+{
+    "version": "90.0.4430.72",
+    "description": "Fast, secure, and free web browser, built for the modern web.",
+    "homepage": "https://www.google.com/chrome/",
+    "license": {
+        "identifier": "Freeware",
+        "url": "https://www.google.com/chrome/privacy/eula_text.html"
+    },
+    "architecture": {
+        "64bit": {
+            "url": "https://dl.google.com/release2/chrome/VYWR7z03VITSenbdO3GEAQ_90.0.4430.72/90.0.4430.72_chrome_installer.exe#/dl.7z",
+            "hash": "704a58c4ccb49d82eea8c6c8354621c9c481fdb0bbc3a070c3d22697475a6332"
+        },
+        "32bit": {
+            "url": "https://dl.google.com/release2/chrome/QXfLrKD3khekJ96AJ562-g_90.0.4430.72/90.0.4430.72_chrome_installer.exe#/dl.7z",
+            "hash": "d29a4c3b994a1f14fec3a67bb44789fe6d44fb0dcc7b7a4a0239f2fa54abadc5"
+        }
+    },
+    "installer": {
+        "script": "Expand-7zipArchive \"$dir\\chrome.7z\" -ExtractDir 'Chrome-bin' -Removal"
+    },
+    "bin": "chrome.exe",
+    "shortcuts": [
+        [
+            "chrome.exe",
+            "Google Chrome"
+        ]
+    ],
+    "checkver": {
+        "url": "https://chrome-dl.com/api/chrome.min.xml",
+        "regex": "(?sm)<stable32><version>(?<version>[\\d.]+)</version>.+release2/chrome/(?<32>[\\w-]+)_.+<stable64>.+release2/chrome/(?<64>[\\w-]+)_.+</stable64>"
```

Comment:
If this is a hand-written regex, let me know and I'll spend the time reviewing it. If it's been downloaded from an upstream then I'll ignore

Reply:
Yeah, it's just copied from the original version that's always using `latest` as I understand it. So this is just a copy of the old version with the version, shas and download path updated.